.github/workflows/release.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - main
       - master
-    tags: ["v.*"]
+    tags: ["v*"]
```

Comment:
in some other places we even have `tags: [ "*" ]`

Reply:
Tags might be useful for other things than releases.